docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,176 @@
+| ***C-Chain*** | [**C-Chain : Export Avax to X-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-xchain-avax.ts) | _C Chain >> X Chain_ |
+| ***C-Chain*** | [**X-Chain : Import Avax from C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildImportTx-cchain.ts) | _C Chain >> X Chain_ |
+| ***C-Chain*** | [**C-Chain : Export Avax to P-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/evm/buildExportTx-pchain.ts) | _C Chain >> P Chain_ |
+| ***C-Chain*** | [**P-Chain : Import Avax from C-Chain**](https://github.com/ava-labs/avalanchejs/blob/master/examples/platformvm/buildImportTx-CChain.ts) | _C Chain >> P Chain_ |
+
+
+### Step 2 - Modify Your Export
+
+:::info
+
+The script in this tutorial uses **X Chain <-> C Chain** transfers. Depending on which chains you want to interact with, the scripts you use could be different. Please check the table above for more information.
+
+:::
+
+Select the **examples/avm** folder to view the AvalancheJS X-Chain examples. To send AVAX from the X-Chain to the C-Chain, select [```avm/buildExportTx-cchain-avax.ts```](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts)
+
+![Image for post](/img/ajs-export-cchain-res.png)
+
+#### Modify your Avalanche network configuration
+
+If you are making calls to your Node from another machine, or started your node using [Advanced Configurations](https://docs.avax.network/nodes/build/set-up-node-with-installer#advanced-node-configuration) you might use a custom IP
+```js
+const ip: string = "localhost" | "custom" 
+```
+
+```js
+const port: number = 9650
+```
+```js
+const protocol: string = "http" | "https"
+```
+
+Depending on the networkID which is passed in when instantiating Avalanche the encoded addresses will have a distinctive HRP per each network.
+```js
+const networkID: number =
+  0: "custom"
+  1: "avax"
+  2: "cascade"
+  3: "denali"
+  4: "everest"
+  5: "fuji"
+  1337: "custom"
+  12345: "local"
+  ```
+
+Example:
+```js
+const ip: string = "localhost"
+const port: number = 9650
+const protocol: string = "http"
+const networkID: number = 5
+```
+
+#### Edit The Amount Of AVAX You Want To Send:
+By default the script sends the wallet's entire AVAX balance:
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance)
+const amount: BN = balance.sub(fee)
+```
+
+
+
+Change the amount by creating a new _BN_ variable: ```value``` and assigning it a string value ```"10000000000000000"``` (.01 AVAX) 
+
+```js
+  const value: BN = new BN("10000000000000000")
+  const amount: BN = value.sub(fee)
+  ```
+
+We must pass the value in WEI format. For reference, 10 ** 18 WEI = 1 AVAX
+
+You can use the [snowtrace unit converter](https://snowtrace.io/unitconverter) to view more unit conversions
+
```

Comment:
Add a section that points out which address as the actual AVAX balance and is paying the fee. Deep link to [this line](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts#L30) in the export script and point out that `DefaultLocalGenesisPrivateKey` is [an AvalancheJS constant](https://github.com/ava-labs/avalanchejs/blob/master/src/utils/constants.ts#L134-L135) and this is the bech32 address which is derived from that private key: `X-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p`

That key is pre-funded on the local network when using Avalanche Network Runner. It has the following AVAX in the genesis block/vertex of each chain on the primary subnet

* X-Chain: 300000000000000000 nAVAX
* P-Chain: 30000000000000000 nAVAX
* C-Chain: 50000000000000000000000000 nAVAX

Reply:
Added Earlier

https://github.com/ava-labs/avalanche-docs/blob/ebc232b5fa0f117a69328a08c3a7142613b6c13a/docs/quickstart/cross-chain-transfers.md?plain=1#L56